src/RazorLight/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -30,6 +36,79 @@ public static void AddRazorLight(this IServiceCollection services, Func<IRazorLi
 			});
 		}
 
+		public static RazorLightDependencyBuilder AddRazorLight(this IServiceCollection services)
+		{
+			services = services ?? throw new ArgumentNullException(nameof(services));
+
+			services.TryAddSingleton<PropertyInjector>();
+			services.TryAddSingleton<ICachingProvider, MemoryCachingProvider>();
+			services.TryAddSingleton<RazorSourceGenerator>();
+			services.TryAddSingleton<IRazorTemplateCompiler, RazorTemplateCompiler>();
+			services.TryAddSingleton<ITemplateFactoryProvider, TemplateFactoryProvider>();			
+			services.TryAddSingleton<IMetadataReferenceManager, DefaultMetadataReferenceManager>();
+			services.TryAddSingleton<ICompilationService, RoslynCompilationService>();
+
+
+			services.TryAddSingleton<IEngineHandler, EngineHandler>();
+			services.TryAddSingleton<IRazorLightEngine, RazorLightEngine>();
+
+			RazorLightDependencyBuilder builder = new RazorLightDependencyBuilder(services);
+
+			return builder;
+		}
+
+
+
+
+		///// <summary>
+		///// Adds required RazorLightProject
+		///// </summary>
+		///// <param name="services"></param>
+		///// <param name="root">Directory path to the root folder containing your Razor markup files.</param>
+		///// <param name="extension">If you wish, you can use a different extension than .cshtml.</param>
+		//public static void AddRazorLightFileSystemProject(this IServiceCollection services,string root, string extension = null)
```

Comment:
Why is all this code commented out?

Reply:
@jzabroski To be honest i just forgot to delete it. It was a initial way how to inject services but i rewrote everthing so the exact logic is now used inside RazorLightDependecyBuilder wich i find more user friendly. So if the RazorLightDependecyBuilder class is ok those comments should be deleted.